### PR TITLE
Fix Scene performance counters

### PR DESCRIPTION
**Pull request by sebavan**
https://forum.babylonjs.com/t/confuse-about-sceneinstrument-caling-fetchnewframe/40840/2

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/13870/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13870/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13870/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/13870/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/13870/merge
https://gui.babylonjs.com/?snapshot=refs/pull/13870/merge
https://nme.babylonjs.com/?snapshot=refs/pull/13870/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/13870/merge#BCU1XR#0

**Review by carolhmj — APPROVED** [review]

File: packages/dev/core/src/Instrumentation/sceneInstrumentation.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ export class SceneInstrumentation implements IDisposable {
 
             this._onAfterActiveMeshesEvaluationObserver = this.scene.onAfterActiveMeshesEvaluationObservable.add(() => {
                 Tools.EndPerformanceCounter("Active meshes evaluation");
-                this._activeMeshesEvaluationTime.endMonitoring();
+                this._activeMeshesEvaluationTime.endMonitoring(false);
```
[the remark this answers is not in the document]
Comment (sebavan) [reply packages/dev/core/src/Instrumentation/sceneInstrumentation.ts:102]: nope, all good as it happens only once per frame

**Pull request merged**